migrations/20170213221219-newSortByColumnForSubject.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const constants = require('../db/constants');
+
+module.exports = {
+  up: function (qi, Sequelize) {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+    */
+    return qi.sequelize.transaction(() => qi.addColumn('Subjects', 'sortBy', {
+      type: Sequelize.STRING(constants.fieldlen.sortField),
+      allowNull: true,
+      defaultValue: '',
+      validate: {
+        is: /^[0-9a-z_-]*$/i,
+      },
+    }));
+  },
+
+  down: function (qi, Sequelize) {
```

Comment:
I hope the changeColumn method would work. The UP and DOWN states now reflect the new and previous state correspondingly. Pls review. Thanks